### PR TITLE
fix: clr prediction estimates showing incorrect value

### PR DESCRIPTION
**Pull request by thelostone-mc**
## Description

- fixes the issue with regard to USDT
- ensures prediction diff is always used 

fixes https://github.com/dcgtc/dgrants/issues/594

Untitled

Untitled

### CODE REVIEW

File: dcurve/src/internal/utils.ts [hunk 1]
```diff
@@ -75,8 +75,8 @@ export function getPredictedMatchingForAmount(clrPredictions: GrantPrediction, a
   // predictions should be provided with the same axis
   const contributionsAxis = [0, 1, 10, 100, 1000, 10000];
   // get all predictions for this grant in this round
-  const clrPredictionCurve = clrPredictions?.predictions?.map((prediction, key) => {
-    return key == 0 ? prediction.predictedGrantMatch : prediction.predictionDiff;
```
Comment (thelostone-mc) [inline dcurve/src/internal/utils.ts:79]: @gdixon do you recollect why this change was made ?

**Review by mk1r — APPROVED** [review]
nice!

**Review by jjwoz — APPROVED** [review]
It looks good and functions as expected but should we use === 0?
